packages/agoric-cli/scripts/get-sdk-package-names.js

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env node
+/* global process, Buffer */
+import { spawn } from 'child_process';
+import { basename } from 'path';
+
+const ps = spawn('yarn', ['workspaces', '--silent', 'info'], {
+  stdio: ['ignore', 'pipe', 'inherit'],
+});
+
+// Get Buffers of output.
+const chunks = [];
+ps.stdout.on('data', data => chunks.push(data));
+
+// Wait for the process to exit.
+ps.on('close', code => {
+  if (code !== 0) {
+    throw Error(`yarn info exited with code ${code}`);
```

Comment:
```suggestion
    throw new Error(`yarn info exited with code ${code}`);
```

Reply:
Is there a reason to prefer using `new` in this case?  I don't see how it is superior.

Reply:
It's equivalent and mostly stylistic in this case. I consider to be legacy behavior that some built-ins return a new instance when called as a function. The direction taken by ECMAScript and codified in `class` is to require `new` to create instances of objects.

I'm actually surprised this passed linting as the default behavior for the [eslint built-in `new-cap` rule](https://eslint.org/docs/rules/new-cap#capisnew) is to error when a function starting with an upper case is called without `new`, as it's usually indicative of a mistake.

Reply:
It seems that our codebase does both so since this is a stylistic concern, I will resolve.